test/lib/ufe/CMakeLists.txt

```diff
@@ -103,7 +104,12 @@ foreach(script ${test_script_files})
     add_test(
         NAME ${target}
         WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-        COMMAND ${MAYA_PY_EXECUTABLE} -c "from unittest import main;import maya.standalone;maya.standalone.initialize(name='python');import ${target};main(module=${target});maya.standalone.uninitialize()"
+        COMMAND ${MAYA_PY_EXECUTABLE} -c "from unittest import main; \
+                                          import maya.standalone; \
+                                          maya.standalone.initialize(name='python'); \
+                                          import ${target}; \
+                                          main(module=${target}); \
+                                          maya.standalone.uninitialize()"
```

Comment:
@kxl-adsk @mattyjams  
I recently fixed a bug in PR198 where ctest was falsely reporting python tests pass.

There are multiple issues here ( line 111,112 ):
1- main(module=${target});  by default calls the sys.exit() and therefore `maya.standalone.uninitialize()` never is called.
2- if there is an error or failure in any of these ufe tests, ctest still says that everything passed.

Solution:
1- don't let the sys.exit() happen which simply can be done by passing `exit=False` argument. 
`main(module=${target}, exit=False)`
2- call the sys.exit based on the TestProgram  program result. maya.standalone.uninitialize() should also happen before the sys.exit call.
```
maya.standalone.uninitialize();
testProg = main(module=${target}, exit=False)
sys.exit(not testProg.result.wasSuccessful());
```
Note: Interestingly, I made an error on purpose in one of these tests but to my surprise ctest reported correctly that the test fail. In any case, it still makes sense to exit based on wasSuccessful() result.

See unittest doc:
https://docs.python.org/2/library/unittest.html#unittest.main

See some of the fixes in PR198:
https://github.com/Autodesk/maya-usd/pull/198/commits/b34dc94655c33c791f360a8090a5daedc8eaee97

I am planning of creating a separate PR to fix this. I also think bundling up all these tests into a TestSuite, and then calling unittest.main once would be a better solution.

Reply:
Ahh, thanks @HamedSabri-adsk! I didn't actually change anything here other than breaking up the one long line into multiple lines. Since you're already addressing that in #198, I'll just revert this so that this PR merges cleanly with yours.

Reply:
@mattyjams Nice, thanks for doing that.